Cargo.toml

```diff
@@ -17,7 +17,6 @@ exclude = [
     "benches/payout-stakers",
     "bin/cliain",
     "bin/runtime/src/chain_extension/snarcos-extension",
-    "contracts/snarcos-contract",
```

Comment:
I agree this contract is maybe not the best as an example, but could still be somehow usefull - e.g. compiling it in CI with relations crate to catch any std creeping in? I looked into various ways of checking whether a given crate compiles in a no_std environment and unfortunately it is not that straightforward, e.g. there is no way to capture it in a starightforward unit test :/

Reply:
I would revive it once we have stable building procedures for contracts (hopefully in a public docker image); also @deuszx had kinda strong opinion on that contract (to remove it)

Reply:
To be honest, I didn't look at this from the perspective that @fbielejec is bringing up - i.e. the litmus test. It's true that the zk-apps will be more difficult to keep aligned with the repo here (unless we add some kind of nightly tests to it) but, at the same time, a lot of the interactions between contract <-> chain extension is happening via pattern matching on the usigned ints so it's hard to tell how many errors would keeping the `snarcos-contract` give us.